website/docs/docs/dbt-cloud/cloud-api.md

```diff
@@ -3,30 +3,56 @@ title: "dbt Cloud API"
 id: "cloud-api"
 ---
 
-### Overview
-
-If you are on the `Team` or `Enterprise` dbt Cloud plan, you will have access to the 
-dbt Cloud API.
+## Overview
 
+Accounts on the _Team_ and _Enterprise_ plans can query the dbt Cloud API.
 The dbt Cloud API can be used to:
 
 - Download artifacts after a job has completed 
 - Kick off a job run from an orchestration tool
 - Manage your dbt Cloud account
+- and more
+
+Full reference documentation for the dbt Cloud API can be found [here](/dbt-cloud/api).
+
+## How to Access the API 
+
+dbt Cloud support two types of API Tokens: User tokens and Service Account
```

Comment:
Should this be "supports"?

Reply:
...and that's why we do code review! Thanks @aaronraff - merging :)